packages/core/integration-tests/test/markdown.js

```diff
@@ -1,24 +1,23 @@
 const assert = require('assert');
 const path = require('path');
-const {bundle, assertBundleTree, outputFS} = require('@parcel/test-utils');
+const {bundle, assertBundles, outputFS} = require('@parcel/test-utils');
 
-describe.skip('markdown', function() {
-  it('should support bundling Markdown', async function() {
+describe('markdown', function() {
+  it.only('should support bundling Markdown', async function() {
     let b = await bundle(
       path.join(__dirname, '/integration/markdown/index.md')
     );
 
-    await assertBundleTree(b, {
-      name: 'index.html',
-      assets: ['index.md'],
-      childBundles: [
-        {
-          type: 'png',
-          assets: ['100x100.png'],
-          childBundles: []
-        }
-      ]
-    });
+    await assertBundles(b, [
+      {
+        name: 'index.html',
+        assets: ['index.md']
+      },
+      {
+        type: 'png',
+        assets: ['100x100.png']
+      }
+    ]);
 
     let files = await outputFS.readdir(path.join(__dirname, '/dist'));
```

Comment:
This test fails at this line with 
```shell
FSError: ENOENT: /..../parcel/packages/core/integration-tests/test/dist does not exist
```
 Where exactly is the output supposed to be? Is the the Markdown Plugin I converted not configured properly? Thoughts?

Reply:
You should use the `distDir` variable from `@parcel/test-utils` here instead.